src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -10,8 +10,8 @@ import { derived, type Readable } from 'svelte/store';
 /**
  * All user-enabled tokens matching the selected network or chain fusion.
  */
-const enabledNetworkTokens: Readable<Token[]> = derived(
-	[enabledTokens, selectedNetwork, pseudoNetworkChainFusion],
+export const enabledNetworkTokens: Readable<Token[]> = derived(
```

Comment:
Not sure why we export this dervied store?

Reply:
it slipped in, sorry, removed